xbmc/XBDateTime.h

```diff
@@ -220,7 +220,7 @@ class CDateTime : public IArchivable
   bool IsValid() const;
 
   static void ResetTimezoneBias(void);
-  static CDateTimeSpan GetTimezoneBias(void);
+  static CDateTimeSpan GetTimezoneBias(const bool forceUpdate = false);
```

Comment:
Nitpick: IMO, `const bool` makes no real sense here. The const qualifier on parameters has only one purpose: prevent the function body from modifying the const qualified arguments. Is this your intention?

Reply:
Since you ask - Of course it's not needed. It was done as a "good practice". Another Kodi member I worked with liked constants.  :)

Reply:
I'm also a big fan of `const`, that's why I'm always looking where usage actually makes sense.